torch/fx/experimental/proxy_tensor.py

```diff
@@ -26,6 +26,7 @@
 import operator
 from torch.utils._stats import count
 import logging
+from torch._library.abstract_impl_class import AbstractScriptObject
```

Comment:
nit: I'm kind of regretting the `abstract` name. We're fake-ifying a FakeTensor, so we should probably call these things FakeScriptObject to be consistent with the Tensor terminology.